api/v1alpha1/drplacementcontrol_types.go

```diff
@@ -112,7 +112,7 @@ type DRPlacementControlSpec struct {
 	// Label selector to identify all the PVCs that need DR protection.
 	// This selector is assumed to be the same for all subscriptions that
 	// need DR protection. It will be passed in to the VRG when it is created
-	PVCSelector metav1.LabelSelector `json:"pvcSelector"`
+	PVCSelector metav1.LabelSelector `json:"pvcSelector,omitempty"`
```

Comment:
Should not be allowed empty. If it is allowed, then by default everything in the namespace is protected. This will not work with VolSync (as its PVCs are on the same namespace), and it will not work with applications where all its PVCs should be protected. Kafka is an example of where zookeeper PVCs should NOT be protected.